input/_ExtensionsLayout.cshtml

```diff
@@ -158,11 +180,35 @@
             @author
         </p>
 
+        @if (contributors.Any())
+        {
+            <h2 class="no-anchor">Contributors</h2>
+            <p>
+                <ul class="list-unstyled">
+                    @foreach (var contributor in contributors)
```

Comment:
@pascalberger Should we remove the bots from the listing?

![image](https://github.com/cake-build/website/assets/349188/60be95da-0efa-49eb-a508-e8b77786493c)

Reply:
@nils-a Data comes from [contributors.json](https://github.com/cake-build/website/blob/master/contributors/contributors.json) which is maintained by Addin Discoverer tool. It already filters accounts marked as bots by GitHub and we can additionally maintain a blacklist there (see https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/226).

The screenshot is from my local dev environment and based on an earlier code base before we filtered bots. In the meantime bots were already removed (see https://github.com/cake-build/website/pull/2714).